test/suites/alpha/drift/suite_test.go

```diff
@@ -87,7 +87,7 @@ var _ = Describe("Drift", Label("AWS"), func() {
 				},
 			},
 		})
-		env.ExpectSettingsOverriddenLegacy(map[string]string{"featureGates.driftEnabled": "true"})
+		env.ExpectSettingsOverridden(v1.EnvVar{Name: "FEATURE_GATES", Value: "Drift=true"})
```

Comment:
Is this right? Shouldn't we be testing if the configmap approach works?

Reply:
Good callout. I'm opting to remove the setting from the environment variable that is automatically injected by helm when setting a setting like this in the ConfigMap. That'll ensure that we are testing the override logic and things are properly being passed through.